wesnoth-thot.po

```diff
@@ -1701,11 +1676,17 @@ msgid ""
 "our superiority as the true people of Irdya, an image that the other races "
 "shall respect and fear!"
 msgstr ""
+"Jako zbraň? Kdepak obřadníku... mé úmysly jsi špatně pochopil. Stejně "
```

Comment:
není to básnička, písnička ani německá věta - má to být české sdělení (vizte [slovosled](https://cs.wikipedia.org/wiki/%C4%8Cesk%C3%BD_slovosled) nebo [pořádek](http://nase-rec.ujc.cas.cz/archiv.php?art=6428))
slovosled je už tak přeházený a některá slova dost neobvyklá, takže vzletnosti tu bylo učiněno zadost
ta slovesa na konec věty nepatří
```suggestion
"Jako zbraň? Kdepak obřadníku... mé úmysly jsi pochopil špatně. Stejně "
```